Source/Programs/Improbable.Unreal.CodeGeneration/Templates/UnrealEntityPipelineHeaderGenerator.tt

```diff
@@ -52,6 +52,8 @@ public:
 	void Remove<#= unrealComponent.CapitalisedName #>ComponentOp(const worker::RemoveComponentOp& Op);
 <# } #>
 
+	FORCEINLINE bool IsInitialized() { return bInitialised; }
```

Comment:
Thought about converting `bInitialised` -> `bInitialized` but this code will likely go away anyway.